build/release/README.md

```diff
@@ -1,34 +1,69 @@
+   release/tag`
 
-Afterwards, the release manager/verifier simply needs to run:
-`./build/release/release.sh <TAG>` (from the top-level `lnd` directory) , where
-`<TAG>` is the name of the next release/tag.
+This will then create a directory of the form `lnd-<TAG>` containing archives
+of the release binaries for each supported operating system and architecture,
+and a manifest file containing the hash of each archive.
 
 ## Verifying a Release
 
-With Go 1.13, it's now possible for third parties to verify a release binary.
-Before this release, one had to trust that release manager to build the proper
-binary. With this new system, third parties can now _independently_ run the
-release process, and verify that all the hashes in the final `manifest.txt`
-match up.
+With `go1.13`, it's now possible for third parties to verify release binaries.
+Before this version of `go`, one had to trust the release manager(s) to build the
+proper binary. With this new system, third parties can now _independently_ run
+the release process, and verify that all the hashes of the release binaries
+match exactly that of the release binaries produced by said third parties.
+
+To verify a release, one must obtain the following tools (many of these come
+installed by default in most Unix systems): `gpg`/`gpg2`, `shashum`, and
+`tar`/`unzip`.
+
+Once done, verifiers can proceed with the following steps:
+
+1. Acquire the archive containing the release binaries for one's specific
+   operating system and architecture, and the manifest file along with its
+   signature.
+2. Verify the signature of the manifest file with `gpg --verify
+   manifest-<TAG>.txt.sig`. This will require obtaining the PGP keys which
+   signed the manifest file, which are included in the release notes.
+3. Recompute the `SHA256` hash of the archive with `shasum -a 256 <filename>`,
+   locate the corresponding one in the manifest file, and ensure they match
+   __exactly__.
+
+At this point, verifiers can use the release binaries acquired if they trust
+the integrity of the release manager(s). Otherwise, one can proceed with the
+guide to verify the release binaries were build properly by obtaining `shasum`
+and `go` (matching the same version used in the release):
+
+4. Extract the release binaries contained within the archive, compute their
+   hashes as done above, and note them down.
+5. Ensure `go` is installed, matching the same version as noted in the release
+   notes. 
+6. Obtain a copy of `lnd`'s source code with `git clone
+   https://github.com/lightningnetwork/lnd`.
+7. Proceed to verify the tag with `git verify-tag <TAG>` and compile the
+   binaries from source for the intended operating system and architecture with
+   `LNDBUILDSYS=OS-ARCH ./build/release/release.sh`
+8. Extract the archive found in the `lnd-<TAG>` directory created by the
+   release script and recompute the hash of the release binaries. These should
```

Comment:
> recompute the hash of the release binaries

What hash?

Reply:
the hash of the extracted binaries, maybe add clarification?

Reply:
sha3? md5? sha512/256?

Reply:
Fixed.